spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe UsersController, type: :controller do
+  describe "Get #show" do
+    before do
+      @user = FactoryBot.create(:user)
+      get :show, params: { id: @user.id }
+    end
+
+    it "Validate that show method render to show view" do
```

Comment:
This verb should be in lowercase and since it is starting with `it` it should be with `s` see the following:
`it "validates ..."`